problem_builder/step.py

```diff
@@ -75,40 +83,109 @@ class StepMixin(object):
         scope=Scope.content
     )
 
+    editable_fields = ('display_name', 'show_title',)
+
+    def _(self, text):
```

Comment:
@itsjeyd I would move that `_` into a common mixin - it is duplicated virtually everywhere